app/src/main/java/nl/eduvpn/app/service/PreferencesService.java

```diff
@@ -350,7 +393,7 @@ public AuthState getCurrentAuthState() {
      */
     @NonNull
     public Settings getAppSettings() {
-        Settings defaultSettings = new Settings(Settings.USE_CUSTOM_TABS_DEFAULT_VALUE, Settings.FORCE_TCP_DEFAULT_VALUE);
+        Settings defaultSettings = new Settings(Settings.USE_CUSTOM_TABS_DEFAULT_VALUE, Settings.FORCE_TCP_DEFAULT_VALUE, Settings.FORCE_TCP_DEFAULT_VALUE);
```

Comment:
The last property should be the WireGuard default value.

Reply:
Fixed in https://github.com/eduvpn/android/pull/318/commits/47e511640a29fdfbd56b66abef85aeb051c26fc1